scripts/sync-css.js

```diff
@@ -0,0 +1,32 @@
+// sync-head.js
+const fs = require('fs');
+const path = require('path');
+
+const cssDirectory = './common/css';
+const orderedCSSFilenames = ['common.css', 'highlightjs-theme.css'];
+
+const hugoCSSPath = './website/assets/css/ec-common.css';
+const antoraCSSPath = './antora/supplemental-ui/css/ec-common.css';
+
+const cssComment = `/* ***************************************************
+ * This CSS is autogenerated.
+ *  
+ * Edit /common/css/common.css to make changes, then 
+ * execute \`npm run sync-css\` to update this file.
+*/`;
+
+// Read shared css contents
+const combinedCSSContent = orderedCSSFilenames
+  .map((filename) => fs.readFileSync(path.join(cssDirectory, filename), 'utf8'))
+  .join('\n');
+
+// Prepend the comment to the shared content
+const combinedCSSContentWithComment = cssComment + '\n\n' + combinedCSSContent; + '\n'
+
+// Save the content to Hugo's CSS file
+fs.writeFileSync(hugoCSSPath, combinedCSSContentWithComment, 'utf8');
+
+// Save the content to Antora's CSS file
+fs.writeFileSync(antoraCSSPath, combinedCSSContentWithComment, 'utf8');
```

Comment:
Hmm, this does feel more complicated than what I was imagining.

Reply:
If preferred, I can update this PR that all common css lives in `/website/assets/css/ec-common.css` with `/website/assets/css/custom.css` and `/antora/supplemental-ui/css/custom.css` being for platform specific overrides. I can then update the antora head template to link to `/site.main`, but the fingerprint would be an issue due to the current way the website is built (we build Antora, then Hugo, so we won't know the fingerprint).

Reply:
I was looking at a minimal version and made #70. It's uses (yet more) symlinks, and doesn't do the packing and fingerprinting, but I think maybe it's good enough for now.